fem/estimators.hpp

```diff
@@ -102,37 +103,43 @@ class ZienkiewiczZhuEstimator : public AnisotropicErrorEstimator
        @param integ    This BilinearFormIntegrator must implement the methods
                        ComputeElementFlux() and ComputeFluxEnergy().
        @param sol      The solution field whose error is to be estimated.
+       @param with_coeff Takes value of 1 if we want to consider the coefficient
+               in the bilinear integrator to weight the fluxed.
        @param flux_fes The ZienkiewiczZhuEstimator assumes ownership of this
                        FiniteElementSpace and will call its Update() method when
                        needed. */
    ZienkiewiczZhuEstimator(BilinearFormIntegrator &integ, GridFunction &sol,
-                           FiniteElementSpace *flux_fes)
+                           FiniteElementSpace *flux_fes, int with_coeff = 0)
```

Comment:
In both constructors, here and below, I'd use a separate method for controlling the `with_coeff` option, similar to `SetAnisotropic()` and `SetFluxAveraging()`.

Reply:
Do you mean to take it out of the constructor and use an option `SetWithCoeff()` instead? As in 
```
void SetWithCoeff() { _with_coeff = 1; }
```
Honestly `_with_coeff` should be a bool instead, I will change that.

Reply:
It think it may be better to allow both - turning the option on or off - e.g. using this signature (similar to `SetAnisotropic`):
```c++
   void SetWithCoeff(bool with_coeff = true) { _with_coeff = with_coeff; }
```

Reply:
Included